### PR TITLE
1 +imei 359090044913811 / 01

### PR DESCRIPTION
**Pull request by ilekoglu**
imei
1+imei 359090044913811 / 1
1+imei 359090044913812 / 1

### CODE REVIEW

**Comment by steinbrueckri** [comment]
WTF is this? Why do you open an issue with the IMEI number of a SAMSUNG Galaxy S9+? :D

**Comment by ilekoglu** [comment]
> 

- - [359090044913810 / 01 ]

File: imei [hunk 1]
```diff
@@ -0,0 +1,3 @@
+imei 357630090327852
+
```
Comment (ilekoglu) [inline imei:2]: 1+imei 359090044913811 / 01

1+imei 359090044913812 / 01

**Review by ilekoglu — COMMENTED** [review]
359090044913811 / 01

File: imei [hunk 2]
```diff
@@ -0,0 +1,3 @@
+imei 357630090327852
```
Comment (ilekoglu) [inline imei:1]: 1 +imei 357630090327853  / 1

2+imei  357630090327854  / 1

**Comment by ilekoglu** [comment]
>  
> 1+imei 359090044913811

**Review by ilekoglu — COMMENTED** [review]
359090044913811

**Comment by ilekoglu** [comment]
>  
> 1+imei 359090044913811 / 1

**Comment by ilekoglu** [comment]
>  
> 1+imei 359090044913811 / 1
> 1+imei 359090044913812 / 2

**Pull request closed without merging**